TCIABrowser/TCIABrowser.py

```diff
@@ -934,9 +936,11 @@ def addSelectedToDownloadQueue(self):
       self.downloadQueue[selectedSeries] = [downloadFolderPath, self.fileSizes[row].text()]
       self.seriesRowNumber[selectedSeries] = row
 
-###
-
-###        
+    # Warn users of the download size and series
+    downloadWarning = f"You have selected {len(self.downloadQueue)} series to download, the download size is {round(imageSizeToDownload, 2)} MB, do youu wish to proceed?"
```

Comment:
There's a typo ("youu") in the warning.  Also, I suspect in many cases people will exceed downloads of 999 megabytes.  Is there a way to get it to switch the warning to express the size in gigabytes if that occurs?  If that's a pain, perhaps it's best to just always display the size in gigabytes.

Reply:
No problem at all, I just changed the warning message to display in either MB or GB, depending on the total download size. If it exceeds 1024 MB, it will switch to display in GB, otherwise, it will stay in MB.